apps/studio/components/interfaces/SQLEditor/DownloadSnippetModal.tsx

```diff
@@ -0,0 +1,150 @@
+
+  const [selectedView, setSelectedView] = useState<'CLI' | 'NPM'>('CLI')
+
+  return (
+    <Modal size="xlarge" header="Download snippet" hideFooter closable {...props}>
+      <div className="flex flex-col items-start justify-between gap-4 px-6 py-3 mb-4">
+        <p className="text-sm mt-2">
+          You can download this snippet as a local{' '}
+          <a
+            className="text-brand"
+            href="https://supabase.com/docs/guides/cli/local-development#database-migrations"
+            target="_blank"
+            rel="noopener"
+          >
+            migration
+          </a>{' '}
+          file via the{' '}
+          <a
+            className="text-brand"
+            href="https://supabase.com/docs/guides/cli/getting-started"
+            target="_blank"
+            rel="noopener"
+          >
+            Supabase CLI
+          </a>
+          .
+        </p>
+        <Separator />
+      </div>
+
+      <TabsProvider>
+        <div className="flex flex-col items-start justify-between gap-4 px-6 py-3 relative mb-6">
+          <div className="absolute top-1 right-6">
+            <TwoOptionToggle
+              width={75}
+              options={['CLI', 'NPM']}
+              activeOption={selectedView}
+              borderOverride="border-gray-100"
+              onClickOption={() =>
+                selectedView === 'CLI' ? setSelectedView('NPM') : setSelectedView('CLI')
+              }
+            />
+          </div>
+
+          <Tabs type="underlined">
+            <Tabs.Panel id="migration" label="Migration">
+              <div className="flex flex-col gap-2 my-2 w-full">
+                <h2 className="text-lg">Download as migration</h2>
+                <p className="text-sm text-scale-1000">
+                  Use the snippet in a new migration named{' '}
+                  <CodeBlock language="bash">{migrationName}</CodeBlock>:
+                </p>
+              </div>
+
+              {selectedView === 'CLI' ? (
+                <CodeBlock
+                  language="bash"
+                  className="language-bash prose dark:prose-dark max-w-none"
+                >
+                  {generateMigrationCliCommand(id, migrationName)}
+                </CodeBlock>
+              ) : (
+                <pre>
+                  <CodeBlock
+                    language="bash"
+                    className="language-bash prose dark:prose-dark max-w-none"
+                  >
+                    {generateMigrationCliCommand(id, migrationName, true)}
+                  </CodeBlock>
+                </pre>
+              )}
+            </Tabs.Panel>
+            <Tabs.Panel id="seed" label="Seed file">
+              <div className="flex flex-col gap-2 my-2 w-full">
+                <h2 className="text-lg">Download as seed file</h2>
+                <p className="text-sm text-scale-1000">
+                  Alternatively if your query consists of sample data, append the snippet to the end
```

Comment:
am just wondering if this should make the outcome clearer

```suggestion
                  Alternatively if your query consists of sample data, run the following command to append the snippet to the end
```

Reply:
I don't think we should add "Alternatively" here either - we should make each tab mutually exclusive i think language wise, although I get what we're trying to do here where the user starts from the first tab, then goes to the second